### PR TITLE
chore: add Peter Ime

### PR DESCRIPTION
**Pull request by PeroHub**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1001"></a>

### CODE REVIEW

**Review by ssensalo — CHANGES_REQUESTED** [review]
Please remove the package files

File: public/data/PeroHub.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      "title": "Started Data Structures & Algorithms",
+      "date": "January 2022",
+      "icon": "Freecodecamp",
+      "color": "red",
+      "description": "Slowly getting better at solving algorithm problems"
+    },
+    {
+      "title": "Started Open Source",
+      "date": "January 2022",
+      "icon": "GitHub",
```
Comment (ssensalo) [inline public/data/PeroHub.json:29]:
```suggestion
      "icon": "github",
```

File: public/data/PeroHub.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      "color": "green",
+      "description": "Started Contibuting To Beginner friendly Issues"
+    },
+    {
+      "title": "Block-Chain Development",
+      "date": "January 2022",
+      "icon": "Globe",
```
Comment (ssensalo) [inline public/data/PeroHub.json:36]:
```suggestion
      "icon": "globe",
```

File: public/data/PeroHub.json [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/Wuzpa",
+      "icon": "twitter"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started Data Structures & Algorithms",
+      "date": "January 2022",
+      "icon": "Freecodecamp",
```
Comment (ssensalo) [inline public/data/PeroHub.json:22]:
```suggestion
      "icon": "freecodecamp",
```

**Review by ssensalo — CHANGES_REQUESTED** [review]

**Review by Vyvy-vi — CHANGES_REQUESTED** [review]
refer to changes suggested in:
- https://github.com/EddieHubCommunity/LinkFree/pull/1001#pullrequestreview-854566373
- https://github.com/EddieHubCommunity/LinkFree/pull/1001#discussion_r786123386
- https://github.com/EddieHubCommunity/LinkFree/pull/1001#discussion_r786123720
- https://github.com/EddieHubCommunity/LinkFree/pull/1001#discussion_r786123958

File: public/data/PeroHub.json (hunk 1 above)
Comment (PeroHub) [reply public/data/PeroHub.json]: I have made changes to this request.

File: public/data/PeroHub.json (hunk 2 above)
Comment (PeroHub) [reply public/data/PeroHub.json]: I have made changes to this request also.

File: public/data/PeroHub.json (hunk 3 above)
Comment (PeroHub) [reply public/data/PeroHub.json]: I have made changes to this request also.

**Comment by PeroHub** [comment]
@ssensalo Please have a look at this let me know if I'm good.

File: src/Components/Profile.js [hunk 4]
```diff
@@ -27,7 +27,6 @@ function Profile({ profile, username }) {
           <p className="text-2xl font-bold mx-2 my-0">({username})</p>
         </div>
         <ShareProfile username={username} />
```
Comment (krishguptadev) [inline src/Components/Profile.js:29]:
```suggestion
        <ShareProfile username={username} />

```

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Review by schmelto — APPROVED** [review]

**Review by nathgoutam93 — APPROVED** [review]

**Pull request merged**